<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMain</class>
 <widget class="QMainWindow" name="AdminMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/evaluation_cropped.png</normaloff>:/images/evaluation_cropped.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_prof">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/images/admin_prof_image.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>white</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #A93016;
border-radius: 10px;
background-color: #000000;
font: 12px;
color: white;
</string>
    </property>
    <property name="text">
     <string>Add New Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #A93016;
border-radius: 10px;
background-color: #000000;
font: 12px;
color: white;</string>
    </property>
    <property name="text">
     <string>Add New Company</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_field">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #A93016;
border-radius: 10px;
background-color: #000000;
font: 12px;
color: white;</string>
    </property>
    <property name="text">
     <string>Add New Field of Action</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subj">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #A93016;
border-radius: 10px;
background-color: #000000;
font: 12px;
color: white;</string>
    </property>
    <property name="text">
     <string>Add New Subject</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>343</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #A93016;</string>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="styleSheet">
     <string notr="true">background: #A93016;</string>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
